app/src/main/java/org/simple/clinic/deeplink/DeepLinkUpdate.kt

```diff
@@ -18,18 +18,19 @@ class DeepLinkUpdate : Update<DeepLinkModel, DeepLinkEvent, DeepLinkEffect> {
       model: DeepLinkModel,
       event: UserFetched
   ): Next<DeepLinkModel, DeepLinkEffect> {
-    return when {
-      event.user?.loggedInStatus == User.LoggedInStatus.LOGGED_IN -> {
-        val effect = if (model.patientUuid != null) {
+    return if (event.user?.loggedInStatus == User.LoggedInStatus.LOGGED_IN) {
+      val effect = if (event.user.capabilities == User.Capabilities(User.CapabilityStatus.Yes)) {
```

Comment:
This check is non-obvious and will get broken when we need to introduce a new capability. We should be explicitly checking the `canTeleconsult` property like

```kotlin
event.user.capabilities.canTeleconsult == CapabilityStatus.Yes
```